docs/syntax_and_semantics/c_bindings/enum.md

```diff
@@ -48,11 +48,11 @@ The type of an enum member is `Int32` by default, even if you specify a differen
 ```crystal
 lib X
   enum SomeEnum
-    A = 1_u32
+    A = 1_u32 # => Error: enum value must be an Int32
```

Comment:
**issue:** I cannot add a line comment there, but the line before this code block needs to be updated as well.

Reply:
Sorry confused as to what you want me to do with this? I just put the comment here to denote the error that gets thrown.

Reply:
Yeah, the comment is good (althoug the `=>` could be removed, we usually use that to designate the value of an expression, not for errors).

I'm referring to line 46. The statement is incorrect and needs an update like this:

```diff
-The type of an enum member is `Int32` by default, even if you specify a different type in a constant value
+The type of an enum member is `Int32` by default.  It's an error to specify a different type in a constant value.
```

Pardon, I shouldn't have made this a line comment at all, it just causes confusion.